.github/workflows/ci.yml

```diff
@@ -58,8 +66,8 @@ jobs:
 
       - name: Install dependencies
         run: |
-          python -m pip install --upgrade pip
-          pip install -r test-requirements.txt
+          pip install --upgrade pip
+          pip install flake8
```

Comment:
We are also installing flake8-bugbear (and flake8-pyi, which shouldn't matter). To avoid this kind of discrepancies, I'd prefer to continue using test-requirements.txt here, even if it installs some unneeded packages.

Reply:
I see! I'll add caching here instead then 🙂